### PR TITLE
SIR does not retain COW items instead Litigation Hold.

### PR DESCRIPTION
**Pull request by sujnai**
Here is the statement from the article:
Also, if a message is changed by a user or a process, copies of the original item are also retained when single item recovery is enabled. 

Here is what we need to update:
Also, if a message is changed by a user or a process, copies of the original item are also retained when single item recovery and Litigation Hold both are enabled.

This is applicable for both Exchange Server 2013, 2016, 2019 and Exchange Online. All reference articles for Single Item Recovery needs to be updated for related products.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ mtps_version: v=EXCHG.150
...
+You can use the Shell to enable or disable single item recovery on a mailbox. In Exchange Server, single item recovery is disabled when a mailbox is created. If single item recovery is enabled, messages that are permanently deleted (purged) by the user are retained in the Recoverable Items folder of the mailbox until the deleted item retention period expires. This lets an administrator recover messages purged by the user before the deleted item retention period expires. Also, if a message is changed by a user or a process, copies of the original item are also retained when single item recovery and Litigation Hold both are enabled.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]:
```suggestion
You can use the Shell to enable or disable single item recovery on a mailbox. In Exchange Server, single item recovery is disabled when a mailbox is created. If single item recovery is enabled, messages that are permanently deleted (purged) by the user are retained in the Recoverable Items folder of the mailbox until the deleted item retention period expires. This lets an administrator recover messages purged by the user before the deleted item retention period expires. Also, if a message is changed by a user or a process, copies of the original item are also retained when both single item recovery and Litigation Hold are enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

File: Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md (hunk 1 above)
Comment (sujnai) [reply Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]: Looks good.

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]: @sujnai could you please commit my change? Thanks.

Comment (sujnai) [reply Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]: The change looks fine @JohanFreelancer9 .

Comment (sujnai) [reply Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]: @JohanFreelancer9 I can see there are other versions of Article for Exchange 2016 and Exchange 2019 as well which needs to be updated with same correction. Should there be a separate pull request created?
https://docs.microsoft.com/en-us/Exchange/recipients/user-mailboxes/single-item-recovery?view=exchserver-2019
https://docs.microsoft.com/en-us/Exchange/recipients/user-mailboxes/single-item-recovery?view=exchserver-2016

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Exchange/ExchangeServer2013/enable-or-disable-single-item-recovery-exchange-2013-help.md:18]: @sujnai - yes, separate pull requests should be created for them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**